src/lib/seam/components/ClimateSettingScheduleTable/ClimateSettingScheduleDetails.tsx

```diff
@@ -0,0 +1,63 @@
+import { DateTime } from 'luxon'
+import type { ClimateSetting, ClimateSettingSchedule } from 'seamapi'
+
+import { useDevice } from 'lib/seam/devices/use-device.js'
+import { DotDivider } from 'lib/ui/layout/DotDivider.js'
+import { ClimateSettingStatus } from 'lib/ui/thermostat/ClimateSettingStatus.js'
+import { useIsDateInPast } from 'lib/ui/use-is-date-in-past.js'
+
+export function ClimateSettingScheduleDetails(props: {
+  climateSettingSchedule: ClimateSettingSchedule
+}): JSX.Element {
+  const { climateSettingSchedule } = props
+  const { device } = useDevice({ device_id: climateSettingSchedule.device_id })
+
+  return (
+    <div className='seam-climate-setting-schedule-details'>
+      <span className='seam-device-name seam-truncated-text'>
+        {device?.properties.name}
+      </span>
+      <DotDivider />
+      <Duration climateSettingSchedule={climateSettingSchedule} />
+      <DotDivider />
+      <ClimateSettingStatus
+        climateSetting={climateSettingSchedule as ClimateSetting}
```

Comment:
Why is this type caseted?

Reply:
https://github.com/seamapi/react/pull/348/commits/993fe636f4ba6d9cafa227d1a246853ac66045fe